src/app/(proper_react)/redesign/(public)/HeroImage.tsx

```diff
@@ -0,0 +1,795 @@
+        ></path>
+        <path
+          fill="url(#paint3_linear_1942_12276)"
+          d="M152.242 90.385l-4.005-2.294-5.587-3.197a4.308 4.308 0 00-4.16-.063l-2.795 1.485a1.396 1.396 0 011.38 0l12.862 7.366c.425.245.688.697.69 1.188v14.732a1.37 1.37 0 01-.693 1.189l-4.351 2.492.679 1.085a2.843 2.843 0 003.827.954l2.153-1.231a4.796 4.796 0 002.418-4.165V94.543a4.793 4.793 0 00-2.418-4.158z"
+        ></path>
+        <path
+          fill="url(#paint4_linear_1942_12276)"
+          d="M150.627 94.874v7.147h4.033v-7.478a4.793 4.793 0 00-2.418-4.158l-4.005-2.294-5.587-3.197a4.308 4.308 0 00-4.16-.063l-2.795 1.485a1.4 1.4 0 011.38 0l12.862 7.366c.426.246.689.7.69 1.192z"
+        ></path>
+        <path
+          fill="url(#paint5_linear_1942_12276)"
+          d="M150.625 103.171v6.431c0 .492-.265.945-.692 1.189l-4.351 2.491.678 1.086a2.846 2.846 0 003.828.954l2.153-1.232a4.795 4.795 0 002.418-4.164v-6.755h-4.034z"
+          opacity="0.9"
+        ></path>
+        <path
+          fill="url(#paint6_linear_1942_12276)"
+          d="M145.274 112.815l-1.887-2.845a10.504 10.504 0 01-3.463 2.057l2.007 3.028a89.272 89.272 0 003.343-2.24z"
+          opacity="0.9"
+        ></path>
+        <text
+          fill="#000"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
+          fontSize="29.225"
+          fontWeight="bold"
+          letterSpacing="-0.209"
+        >
+          <tspan x="169.275" y="112.431">
+            Mozilla
+          </tspan>
+        </text>
+        <text
+          fill="#000"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-metropolis)"
+          fontSize="29.225"
+          letterSpacing="-0.626"
+        >
+          <tspan x="276.115" y="112.431">
+            Monitor
+          </tspan>
+        </text>
+        <path
+          fill="#fff"
+          d="M47.502 161.79a5.65 5.65 0 015.65-5.649H448.89a5.649 5.649 0 015.65 5.649v486.072a5.65 5.65 0 01-5.65 5.65H53.151a5.65 5.65 0 01-5.649-5.65V161.79z"
+        ></path>
+        <path
+          stroke="#000"
+          strokeOpacity="0.1"
+          strokeWidth="0.706"
+          d="M53.152 155.788a6.002 6.002 0 00-6.003 6.002v486.072a6.003 6.003 0 006.003 6.003H448.89a6.003 6.003 0 006.003-6.003V161.79a6.002 6.002 0 00-6.003-6.002H53.151z"
+        ></path>
+        <text
+          fill="#6D6D6E"
+          style={{ whiteSpace: "pre" }}
+          fontFamily="var(--font-inter)"
+          fontSize="16.344"
+          letterSpacing="0"
+        >
+          <tspan x="161.529" y="536.999">
+            {l10n.getString("passwords")}
```

Comment:
These are the same strings as used in the UI. Usually we try not to re-use the same strings in different places, but in this case, this is supposed to be a screenshot of the actual UI, and thus the format (capitalisation, etc.) of these strings should be the same as in the actual UI.